demo/src/demos/transports/index.js

```diff
@@ -0,0 +1,117 @@
+// @flow
+import React, { Component } from "react";
+import TransportWebUSB from "@ledgerhq/hw-transport-webusb";
+import TransportWebBLE from "@ledgerhq/hw-transport-web-ble";
+import styled from "styled-components";
+import axios from "axios";
+import { setNetwork } from "@ledgerhq/live-common/lib/network";
+import genuineCheck from "@ledgerhq/live-common/lib/hw/genuineCheck";
+import getDeviceInfo from "@ledgerhq/live-common/lib/hw/getDeviceInfo";
+
+setNetwork(axios);
+
+let queue = Promise.resolve();
+const execInQueue = <T>(job: () => Promise<T>) => {
+  const p = queue.then(job);
+  queue = p;
```

Comment:
don't need to use `p` ref, but I guess it's for clarity

Reply:
(btw this is demo we dont care)

Reply:
it actually is required for flow last time i checked.

Reply:
p is of time `Promise<T>` so flow needs to remember that.

queue is a `Promise<any>` which accepts `Promise<T>`